.env

```
@@ -0,0 +1,76 @@
+APP_NAME="OmniPost CMS"
+APP_ENV=local
+APP_KEY=base64:crxCfwz6MBQQSJwQPuYR8VQ3ZWfkEoY4kmHGlivnxKE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_KEY key should go before the APP_NAME key

(UnorderedKey)